src/app/features/dialogs/transaction-action-dialog/cancel-stx-transaction-dialog.tsx

```diff
@@ -0,0 +1,18 @@
+import { RouteUrls } from '@shared/route-urls';
+
+import { IncreaseFeeField } from './components/increase-fee-field';
+import { useStxCancelTransaction } from './hooks/use-stx-cancel-transaction';
+import { StxTransactionActionDialog } from './stx-transaction-action-dialog';
+
+export function CancelStxTransactionDialog() {
+  return (
+    <StxTransactionActionDialog
+      title="Cancel transaction"
+      description="Canceling a transaction isn't guaranteed to work. A higher fee can help replace the old transaction"
```

Comment:
`Canceling` is not spelt correctly here (in standard British English)

Did you get the copy text from https://github.com/leather-wallet/extension/issues/3406?

Reply:
> `Canceling` is not spelt correctly here (in standard British English)
> 
> Did you get the copy text from #3406?

Yes i did , i copied the text from one of the comments suggesting the workflow. If there is a new description which is  better i can replace it along with the above suggested changes